lib/mercadopago/client.rb

```diff
@@ -142,6 +144,27 @@ def search(search_hash)
     end
 
     #
+    # Performs a generic GET to the given URL
+    #
+    # - url: the URL to request
+    #
+    def get(url)
+      MercadoPago::Request.wrap_get("#{url}?access_token=#{@access_token}")
```

Comment:
@etagwerker , why do you think that's needed? `wrap_get` [does not accept](https://github.com/ombulabs/mercadopago/blob/master/lib/mercadopago/request.rb#L34-L36) a payload. If you want this method to accept a payload, `wrap_get` should be modified as well. In addition, the official MP gem does not support a payload either.